spec/integration/manifest.spec.yml

```diff
@@ -410,6 +410,24 @@ spec:
                 - access
           deployment_modes:
             $ref: "#/definitions/deployment_modes"
+          links:
```

Comment:
I think some of the fields defined under `links` should be set as required to ensure all the information required by Kibana is present.


```yaml
      links:
        description: ...
        type: array
        minItems: 1
        items:
          type: object
          properties: 
            - ...
          required:
            - title
            - link
            - url
```
Not sure if it could be leave some as not required. At least in the examples, there is one entry without the `content` field.

Reply:
Yes I think content could be optional. It would be the description of the link.